v2/server/controllers/users/sendInvitation.js

```diff
@@ -44,9 +42,8 @@ const SendInvitation = async (req, res, next) => {
         date,
         type,
         trainerName,
-        location: region,
-        startTime,
-        endTime,
+        startTime: startTime || 'N/A',
```

Comment:
Hi, @RamyAlshurafa  I can see that you delete region from stored email data is that intended or by mistake?!